www/src/components/shared/footer-links.js

```diff
@@ -18,16 +18,15 @@ const FooterList = styled.ul`
 
   li {
     display: inline-block;
-    margin-right: 1em;
 
-    &:after {
-      color: ${colors.grey[30]};
-      content: "•";
-      padding-left: 1em;
-    }
+    &:not(:last-of-type) {
```

Comment:
`:not` is a selector you should try to avoid. It's a pretty heavy selector as the css parser has to get all elements and start excluding elements that match the not pattern.

Reply:
Thanks a lot Ward, I didn't know about this behavior.

Reply:
It's how the css parser works in the browser. It's not super bad as the css parser is super fast but if you don't need it, don't use it 😄.

Thanks for updating so fast!